test-alerts.cpp

```diff
@@ -2,7 +2,84 @@
 
 #include "test/catch.hpp"
 #include "typewise-alert.h"
+#include "Test-alert.h"
 
-TEST_CASE("infers the breach according to limits") {
-  REQUIRE(inferBreach(12, 20, 30) == TOO_LOW);
+/***Global variables*****/
+
+int Test_GUI = NO_OUTPUT;
```

Comment:
This global is not protected by preprocessor directive... but the code which accesses it is protected.
If someone accidentally writes code to use this in non-test code, it may get complicated.